### ISSUE TITLE
aria-current missing from active nav item

### ISSUE THREAD

**Issue by seanforyou23** [patternfly/patternfly-react-seed#97]
My understanding is that [aria-current for NavLink](https://reactrouter.com/web/api/NavLink/aria-current-string) is automatically set by inspecting the current URL and comparing it against the route path defined for a given route. It doesn't seem to be working, it would be great if we can fix it as it improves the experience for users of assistive technology.

https://www.w3.org/TR/wai-aria-1.1/#aria-current

**Comment by mturley** [comment]
This is definitely weird. Copying over my troubleshooting from Slack:

> We are using react-router-dom's `<NavLink>` there to render an `<a>` that is aware of when it matches the active route. I know it matches the route properly because it correctly assigns the `activeClassName` we give it when it's active. But according to the docs at https://reactrouter.com/web/api/NavLink, it has a `aria-current` prop that defaults to `"page"`, which should also be applied to the `<a>` when it's active, and it just isn't. Setting that prop manually doesn't fix it, either.

> It seemed like a react-router bug, but it works fine in a sandbox: https://codesandbox.io/s/react-router-g98jr
